pyserini/trectools/_base.py

```diff
@@ -198,3 +198,61 @@ def concat(runs):
         run = TrecRun()
         run.run_data = run.run_data.append([run.run_data for run in runs])
         return run
+
+
+class TrecQrels:
+    """Wrapper class for trec qrels.
+
+    Parameters
+    ----------
+    filepath : str
+        File path of a given Trec Qrels.
+    """
+
+    columns = ['topic', 'q0', 'docid', 'relevance']
+
+    def __init__(self, filepath: str = None):
+        self.filepath = filepath
+        self.qrels_data = pd.DataFrame(columns=TrecQrels.columns)
+
+        if filepath is not None:
+            self.read_run(self.filepath)
+
+    def read_run(self, filepath: str):
+        self.qrels_data = pd.read_csv(filepath, sep='\s+', names=TrecQrels.columns)
+
+    def _get_relevance(self) -> Set[str]:
+        """
+            Returns a set with all relevances.
+        """
+
+        return set(sorted(self.qrels_data["relevance"].unique()))
+
+    def topics(self) -> Set[str]:
+        """
+            Returns a set with all topics.
+        """
+
+        return set(sorted(self.qrels_data["topic"].unique()))
+
+    def get_docids(self, relevance=None, topics=None) -> List[str]:
+        """Return a list of docids given the topics & relevance of interest
```

Comment:
""""Return a list of docids for a topic and a relevance grade."""

Reply:
Also, another common method is `get_relevant_docids`, with a particular relevance grade, default >=1.

That is, return all documents graded 1 or above. Please implement this?